deploy/scripts/src/authentication/test.ts

```diff
@@ -64,6 +64,21 @@ const profiles: ProfileList = {
       exec: 'signIn'
     }
   },
+  load: {
+    signIn: {
+      executor: 'ramping-arrival-rate',
+      startRate: 1,
+      timeUnit: '1s',
+      preAllocatedVUs: 1,
+      maxVUs: 900,
```

Comment:
```suggestion
      maxVUs: 6000,
```
The existing load profiles assume 30 second iterations for max VU calculation, so just up this value to `6000`